css/base.css

```diff
@@ -128,10 +128,96 @@ input[type="email"]:focus {
   border: 1px solid #03909b;
   outline: 0;
 }
+/* Font/Icon Styles
+=======================================*/
+@font-face {
+  font-family: 'icomoon';
+  src:url('fonts/icomoon.eot?gyawo2');
+  src:url('fonts/icomoon.eot?#iefixgyawo2') format('embedded-opentype'),
+    url('fonts/icomoon.ttf?gyawo2') format('truetype'),
+    url('fonts/icomoon.woff?gyawo2') format('woff'),
+    url('fonts/icomoon.svg?gyawo2#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'icomoon';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-menu3:before {
+  content: "\e9bf";
+}
+.icon-circle-right:before {
+  content: "\ea42";
+}
+.icon-circle-left:before {
+  content: "\ea44";
+}
+.icon-facebook3:before {
+  content: "\ea8e";
+}
+.icon-twitter3:before {
+  content: "\ea93";
+}
+.icon-pinterest:before {
+  content: "\ead0";
+}
+
+@font-face {
```

Comment:
Please review this: http://www.anotheruiguy.com/ux-design-dev/_book/web-fonts/README.html and revise your font-families